### PR TITLE
[top, xbar] Add register slice to rom_ctrl response

### PR DESCRIPTION
**Pull request by tjaychen**
- The rom return data can be very slow, this breaks up the
  long path from rom into ibex.

### CODE REVIEW

**Comment by tjaychen** [comment]
@arnonsha this is the official fix for the rom timing.

File: util/tlgen/elaborate.py [hunk 1]
```diff
@@ -150,39 +157,49 @@ def process_pipeline(xbar):
...
             elif full_passthru:
-                dnode.hpass = 1
+                dnode.hreq_pass = 1
+                dnode.hrsp_pass = 1
                 dnode.hdepth = 0
 
-            log.info(
-                "Finished processing socket1n {}, pass={}, depth={}".format(
-                    dnode.name, dnode.hpass, dnode.hdepth))
+            print(
```
Comment (eunchan) [inline util/tlgen/elaborate.py:176]: could you revert these back? I think if you add `--verbose` then you can see the INFO logs.

**Review by eunchan — APPROVED** [review]

File: util/tlgen/elaborate.py (hunk 1 above)
Comment (tjaychen) [reply util/tlgen/elaborate.py:176]: ah yeah sorry, this one is meant to be rebased away. If you look at #11624 you'll see that I reverted already.

Comment (tjaychen) [reply util/tlgen/elaborate.py]: alright! i rebased.

**Review by eunchan — APPROVED** [review]

**Review by msfschaffner — APPROVED** [review]

**Pull request merged**